component-tools/src/test/java/org/talend/sdk/component/tools/DitaDocumentationGeneratorTest.java

```diff
@@ -36,12 +38,16 @@
 import org.junit.jupiter.api.io.TempDir;
 
 import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 @Slf4j
 class DitaDocumentationGeneratorTest extends GeneratorBase {
 
-    @Test
+    // @Test
```

Comment:
Drop the test if no more relevant or fix it please ;)

Reply:
Test kept